openedx/core/djangolib/tests/test_markup.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
 """
-Tests for util.markup
+Tests for openedx.core.djangolib.markup
 """
 
 import unittest
 
 import ddt
+from mako.template import Template
```

Comment:
I used this when I did the js_utils.py because I couldn't get the other odd wrapping logic to work.  I adjusted this to match.  I assumed that edxmako was to help with other challenges and it didn't make sense to use unless it was needed, but that might have been the wrong call?